content/doc/developer/publishing/_chapter.yml

```diff
@@ -3,7 +3,8 @@ sections:
 - style-guides
 - source-code-hosting
 - artifact-repository
-- wiki-page
+- documentation
+# - wiki-page
```

Comment:
Does this work? I vaguely remember problems on pages that don't show in the chapter sections list.

Reply:
It works